app/assets/stylesheets/main.scss

```
@@ -0,0 +1,127 @@
+body {
+  background-color: #BCF4DE;
```

Then reuse the variables like: 
```suggestion
  background-color: $dark-blue;
```